forms-flow-web/src/components/SkeletonLoading/TableSkeleton.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Skeleton from "react-loading-skeleton";
+import "react-loading-skeleton/dist/skeleton.css";
+//takes the number of column and number of rows as props
+const TableSkeleton = React.memo(({ columns, rows, cellHeight = 25}) => {
+  const renderSkeletonCells = (count) =>
+    Array.from({ length: count }).map((_, i) => (
+      <td key={i}>
+        <Skeleton height={cellHeight} />
+      </td>
+    ));
+
+  const renderLastRow = () => (
```

Comment:
It's better to refactor the function, such as renaming it to something like renderLastRowWithPagination, because the current logic is hard to understand. Also, make the pagination count for the skeleton dynamic, with a default value, so it can be easily changed later if needed. Right? @arun-s-aot

Reply:
@Josephalexantony-aot  we can move this into micro front-end to reusable component right  @arun-s-aot

Reply:
Removed the component and uninstalled the package and updated in MF Components